docs/api_reference/requirements.txt

```diff
@@ -2,6 +2,7 @@
 -e libs/langchain
 -e libs/core
 -e libs/community
+-e libs/partners/google-genai
```

Comment:
could also do this as a separate CI step that lists partner packages to avoid maintaining this

Reply:
nvm this is for sphinx